source/development/makefile.rst

```diff
@@ -89,21 +89,21 @@ Available flags
 +-----------------------+------------------+-----------------------------------------------------------------------------------------------------+
 | **V**                 | Display full compiler messages.                                                                                        |
 |                       +------------------+-----------------------------------------------------------------------------------------------------+
-|                       | Default value    | 0                                                                                                   |
+|                       | Default value    | no                                                                                                  |
```

Comment:
Set the default value as `n/a` because the flag is not applied when it is not defined. Same for the others.